src/store/ChartState.js

```diff
@@ -21,6 +22,44 @@ class ChartState {
         this.stxx.addEventListener('layout', this.saveLayout.bind(this));
         this.stxx.addEventListener('symbolChange', this.saveLayout.bind(this));
         this.stxx.addEventListener('drawing', this.saveDrawings.bind(this));
+
+        setTimeout(() => {
+            const masterData = this.stxx.masterData;
+            const lastData = masterData[masterData.length - 1];
+
+            for (let i = 1; i <= 10; i++) {
```

Comment:
I'm not too fond of this hardcoding (and the lack of explanation for doing so). Seems like Morteza will be taking over to review the code here, so I wish you all the best for this.